carina-webdriver/src/main/java/com/qaprosoft/carina/core/foundation/webdriver/decorator/ExtendedWebElement.java

```diff
@@ -229,13 +230,17 @@ public WebElement getElement() {
     }
 
     /**
-     * Reinitializes the element
+     * Reinitializes the element (not applicable to list element)
      *
      * @throws NoSuchElementException if the element is not found
      */
     public void refresh() {
-        // try to override element
-        element = this.findElement();
+        if (isSingleElement) {
+            // try to override element
+            element = this.findElement();
+        } else {
+            LOGGER.debug("Cannot refresh element of the list.");
```

Comment:
how about warn or even error log message?

Reply:
done, changed log level to error